### ISSUE TITLE
Base starter validation fails

### ISSUE THREAD

**Issue by denis-anisimov** [vaadin/base-starter-flow-osgi#107]
PR: https://github.com/vaadin/base-starter-flow-osgi/pull/106

Doesn't make any changes at all (one empty extra line).

It's validation fails: 
```
<i> [ApplicationThemePlugin] Searching themes folder /home/travis/build/vaadin/base-starter-flow-osgi/starter/frontend/themes for theme my-theme
/home/travis/build/vaadin/base-starter-flow-osgi/starter/node_modules/.pnpm/registry.npmjs.org/webpack-cli/3.3.11_webpack@4.42.0/node_modules/webpack-cli/bin/cli.js:281
				throw err;
				^
Error: ENOENT: no such file or directory, mkdir '/home/travis/build/vaadin/base-starter-flow-osgi/starter/target/classes/META-INF/VAADIN/webapp/VAADIN/static/themes/my-theme/components'
    at Object.fs.mkdirSync (fs.js:885:18)
    at fs.readdirSync.forEach.file (/home/travis/build/vaadin/base-starter-flow-osgi/starter/node_modules/@vaadin/application-theme-plugin/theme-copy.js:60:12)
    at Array.forEach (<anonymous>)
    at copyThemeFiles (/home/travis/build/vaadin/base-starter-flow-osgi/starter/node_modules/@vaadin/application-theme-plugin/theme-copy.js:54:32)
    at copyThemeResources (/home/travis/build/vaadin/base-starter-flow-osgi/starter/node_modules/@vaadin/application-theme-plugin/theme-copy.js:41:3)
    at handleThemes (/home/travis/build/vaadin/base-starter-flow-osgi/starter/node_modules/@vaadin/application-theme-plugin/theme-handle.js:121:5)
    at findThemeFolderAndHandleTheme (/home/travis/build/vaadin/base-starter-flow-osgi/starter/node_modules/@vaadin/application-theme-plugin/theme-handle.js:64:23)
    at processThemeResources (/home/travis/build/vaadin/base-starter-flow-osgi/starter/node_modules/@vaadin/application-theme-plugin/theme-handle.js:42:5)
    at compiler.hooks.afterEnvironment.tap (/home/travis/build/vaadin/base-starter-flow-osgi/starter/node_modules/@vaadin/application-theme-plugin/application-theme-plugin.js:49:15)
    at SyncHook.eval [as call] (eval at create (/home/travis/build/vaadin/base-starter-flow-osgi/starter/node_modules/.pnpm/registry.npmjs.org/tapable/1.1.3/node_modules/tapable/lib/HookCodeFactory.js:19:10), <anonymous>:7:1)
    at SyncHook.lazyCompileHook (/home/travis/build/vaadin/base-starter-flow-osgi/starter/node_modules/.pnpm/registry.npmjs.org/tapable/1.1.3/node_modules/tapable/lib/Hook.js:154:20)
    at webpack (/home/travis/build/vaadin/base-starter-flow-osgi/starter/node_modules/.pnpm/registry.npmjs.org/webpack/4.42.0_webpack@4.42.0/node_modules/webpack/lib/webpack.js:56:35)
    at processOptions (/home/travis/build/vaadin/base-starter-flow-osgi/starter/node_modules/.pnpm/registry.npmjs.org/webpack-cli/3.3.11_webpack@4.42.0/node_modules/webpack-cli/bin/cli.js:272:16)
    at yargs.parse (/home/travis/build/vaadin/base-starter-flow-osgi/starter/node_modules/.pnpm/registry.npmjs.org/webpack-cli/3.3.11_webpack@4.42.0/node_modules/webpack-cli/bin/cli.js:364:3)
    at Object.parse (/home/travis/build/vaadin/base-starter-flow-osgi/starter/node_modules/.pnpm/registry.npmjs.org/yargs/13.2.4/node_modules/yargs/yargs.js:567:18)
    at /home/travis/build/vaadin/base-starter-flow-osgi/starter/node_modules/.pnpm/registry.npmjs.org/webpack-cli/3.3.11_webpack@4.42.0/node_modules/webpack-cli/bin/cli.js:49:8
    at Object.<anonymous> (/home/travis/build/vaadin/base-starter-flow-osgi/starter/node_modules/.pnpm/registry.npmjs.org/webpack-cli/3.3.11_webpack@4.42.0/node_modules/webpack-cli/bin/cli.js:366:3)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32)
    at tryModuleLoad (module.js:497:12)
    at Function.Module._load (module.js:489:3)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/travis/build/vaadin/base-starter-flow-osgi/starter/node_modules/.pnpm/registry.npmjs.org/webpack/4.42.0_webpack@4.42.0/node_modules/webpack/bin/webpack.js:156:2)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32)
    at tryModuleLoad (module.js:497:12)
    at Function.Module._load (module.js:489:3)
```

**Comment by denis-anisimov** [comment]
Fixed in Flow.

**Comment by caalador** [comment]
Fixed by vaadin/flow#9812
